test/_init_tests.js

```diff
@@ -91,13 +91,14 @@ afterEach(() => {
   window.ENABLE_LOG = false;
   window.AMP_DEV_MODE = false;
   window.context = undefined;
+  if (sandboxes.length > 0) {
+    sandboxes.splice(0, sandboxes.length).forEach(sb => sb.restore());
```

Comment:
Because you should have already restored them. This is a fallback that prevents the screwup (not restoring) from bringing the rest of the test suite down.